modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -219,6 +219,32 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/ObjectEntry"
             tags: ["ObjectEntry"]
+        patch:
+            operationId: patchByExternalReferenceCode
+            parameters:
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/ObjectEntry"
+                    application/xml:
+                        schema:
+                            $ref: "#/components/schemas/ObjectEntry"
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntry"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectEntry"
+            tags: ["ObjectEntry"]
```

Comment:
Hi @devpedrotavares 

I believe we also need to add one to `"/scopes/{scopeKey}/by-external-reference-code/{externalReferenceCode}":` ([line 294](https://github.com/liferay-objects/liferay-portal/blob/eec7a82dd5ae372dae02d5a2844b56c8cb6cad97/modules/apps/object/object-rest-impl/rest-openapi.yaml#L294)) for objects scoped by site. Could you please check? Thanks!

![image](https://user-images.githubusercontent.com/34186632/213785013-2e22bbfe-fe76-40da-8236-bc114644f725.png)

Reply:
Hi @carolmariaabb,
that is right! We needed to add this endpoint. I also had to override the generated patch to make it call the proper methods.